demo_classification_tf_for_database.py

```diff
@@ -0,0 +1,89 @@
+import tensorflow as tf
+import numpy as np
+import cv2 as cv
+import argparse
+import os
+from os import listdir
+from os.path import isfile, join
+
+
+def sigmoid(x):
+  return 1 / (1 + np.exp(-x))
+
+def load_pb(path_to_pb):
+    with tf.io.gfile.GFile(path_to_pb, "rb") as f:
+        graph_def = tf.compat.v1.GraphDef()
+        graph_def.ParseFromString(f.read())
+        return graph_def
+
+def find_right_val(image_name, list):
+        for text in list: 
+            if image_name in text: 
+                parts = text.rsplit(' ')
+                return int(parts[1])
```

Comment:
For N images we have N*N operations in total. Can we just iterate over this list and predict known images and compare them with the labels?